spec/plugin_spec.rb

```diff
@@ -216,4 +216,14 @@ def create_commit_comment(repo, sha, body, path = nil, line = nil, position = ni
       expect(user.can_review?).to eq(false)
     end
   end
+
+  describe 'code_review_default_parent_category' do
+    fab!(:parent_category) { Fabricate(:category) }
+    fab!(:child_category) { Fabricate(:category, parent_category: parent_category) }
+
+    it 'check if code_review_default_parent_category is valid' do
+      expect { SiteSetting.code_review_default_parent_category = parent_category.id }.not_to raise_error
+      expect { SiteSetting.code_review_default_parent_category = child_category.id }.to raise_error
```

Comment:
We need to specify the error we are expecting otherwise we'll get a warning when running the spec. 

> WARNING: Using the `raise_error` matcher without providing a specific error or message risks false positives, since `raise_error` will match when Ruby raises a `NoMethodError`, `NameError` or `ArgumentError`, potentially allowing the expectation to pass without even executing the method you are intending to call.

Reply:
For completeness, I think we can also add a test case when an invalid category id is assigned.